core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -177,7 +177,7 @@ private[spark] class Executor(
   // Whether to monitor killed / interrupted tasks
   private val taskReaperEnabled = conf.get(TASK_REAPER_ENABLED)
 
-  private val killOnFatalErrorDepth = conf.get(EXECUTOR_KILL_ON_FATAL_ERROR_DEPTH)
+  private val killOnFatalErrorDepth = conf.get(KILL_ON_FATAL_DEPTH)
```

Comment:
Minor: Why did you remove ERROR from KILL_ON_FATAL_DEPTH name?

Reply:
Good catch. Added back ERROR.